hack/gen-swagger-doc/README.md

```diff
@@ -0,0 +1,18 @@
+This folder contains the sources needed to build the gen-swagger-doc container.
+
+To build the container image, 
+
+```
+$ sudo docker build -t gcr.io/google_containers/gen-swagger-docs:v1 .
+```
+
+To generate the html docs,
+
+```
+$ ./run-gen-swagger-docs.sh <API version> <absolute output path, default to PWD>
```

Comment:
What's the general rule for when to use a makefile and when to use a script? There are many scripts in the repo already.